haiku-apps/timebomb/timebomb-0.4.0.recipe

```diff
@@ -0,0 +1,46 @@
+SUMMARY="A loud digital alarm clock"
+DESCRIPTION="TimeBomb is a little customizable alarm clock application. \
+You can change its alarm sound, message, volume, and color!"
+HOMEPAGE="https://github.com/HaikuArchives/TimeBomb"
+COPYRIGHT="1999 Hailstone Software"
+LICENSE="MIT"
+REVISION="1"
+srcGitRev="09ce89ac6e3e028c4e88548b3e850e70bb392eb2"
+SOURCE_URI="https://github.com/HaikuArchives/TimeBomb/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="90393ffbcdc2a7be8ee173423e175af4e378c19eadf91371fd988e4f20ae98a6"
+SOURCE_FILENAME="TimeBomb-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="TimeBomb-$srcGitRev"
+PATCHES="timebomb-$portVersion.patchset"
+
+ARCHITECTURES="x86_gcc2 x86_64"
+
+PROVIDES="
+	timebomb = $portVersion
+	app:TimeBomb = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:g++
+	cmd:make
+	"
+
+BUILD()
+{
+	make -C src $jobArgs OBJ_DIR=objects
+}
+
+INSTALL()
+{
+	mkdir -p $appsDir/TimeBomb
+	cp src/objects/TimeBomb $appsDir/TimeBomb
+	cp -r src/config.file $appsDir/TimeBomb
```

Comment:
If you save the configuration file there, it's impossible to modify it. The correct way should be to make the app saves/load the configuration from `B_USER_SETTINGS_DIR`